From 9e1c3b9f8d891a5761f78ac215584a5530135547 Mon Sep 17 00:00:00 2001
From: Jagpal Singh Gill <paligill@gmail.com>
Date: Thu, 25 Sep 2025 13:36:01 -0700
Subject: [PATCH] meta: ventura: drop valve cable inventory

Drop the ventura valve cable inventory as there is some issue in EM due
to which two configurations with same probe clauses doesn't get probed
appropriately. For now only Valve inventory is required to check valve
presence. Once the actual issue in EM is resolved, the valve cable
inventory item can be added back.

Change-Id: Ia81355e7d0613b6b8df14de80debc92e88c8519b
Signed-off-by: Jagpal Singh Gill <paligill@gmail.com>
---
 .../meta/ventura/ventura_cable.json           | 38 -------------------
 1 file changed, 38 deletions(-)

diff --git a/configurations/meta/ventura/ventura_cable.json b/configurations/meta/ventura/ventura_cable.json
index 7ad2030..161880d 100644
--- a/configurations/meta/ventura/ventura_cable.json
+++ b/configurations/meta/ventura/ventura_cable.json
@@ -305,43 +305,5 @@
             "Length": 0.9,
             "Type": "Copper"
         }
-    },
-    {
-        "Exposes": [],
-        "Name": "$Name",
-        "Probe": [
-            "xyz.openbmc_project.Inventory.Source.DevicePresence({'Name': '^ReturnValve_1$'})"
-        ],
-        "Type": "Cable",
-        "xyz.openbmc_project.Inventory.Decorator.Asset": {
-            "Manufacturer": "Unknown",
-            "Model": "Unknown",
-            "PartNumber": "Unknown",
-            "SerialNumber": "Unknown",
-            "SparePartNumber": "07-100705"
-        },
-        "xyz.openbmc_project.Inventory.Decorator.Cable": {
-            "Length": 0.46,
-            "Type": "Copper"
-        }
-    },
-    {
-        "Exposes": [],
-        "Name": "$Name",
-        "Probe": [
-            "xyz.openbmc_project.Inventory.Source.DevicePresence({'Name': '^SupplyValve_1$'})"
-        ],
-        "Type": "Cable",
-        "xyz.openbmc_project.Inventory.Decorator.Asset": {
-            "Manufacturer": "Unknown",
-            "Model": "Unknown",
-            "PartNumber": "Unknown",
-            "SerialNumber": "Unknown",
-            "SparePartNumber": "07-100704"
-        },
-        "xyz.openbmc_project.Inventory.Decorator.Cable": {
-            "Length": 0.46,
-            "Type": "Copper"
-        }
     }
 ]
-- 
2.47.3

